README.md

```diff
@@ -30,10 +30,11 @@ To protect the data on your drives from falling into the wrong hands during tran
 ### Shared tags
 * `latest`, `${IMAGE_VERSION}`
 
-This image uses semver releases with the major version matching the Truenas API version that is used.
+This image uses the [semver](https://semver.org/) format for releases, with this image's major version matching the Truenas API major version in use.
 
+For example, all 2.*.* image releases use Truenas API v2
 
-**Avoid using the `latest` tag in production environments. Lock the tag to a specific version**
+**Avoid using the `latest` tag in production environments. Lock the tag to a specific version to prevent version conflics**
```

Comment:
```suggestion
**Avoid using the `latest` tag in production environments. Lock the tag to a specific version instead.**
```
There are several reasons not to use latests. Probably doesn't make sense to list any here.